src/layers/normalise.jl

```diff
@@ -139,7 +139,7 @@ testmode!(m::AlphaDropout, mode=true) =
   (m.active = (isnothing(mode) || mode == :auto) ? nothing : !mode; m)
 
 """
-    LayerNorm(sz, λ=identity; affine=true, ϵ=1fe-5)
+    LayerNorm(sz, λ=nothing; affine=true, ϵ=1fe-5)
```

Comment:
Revert to match code:
```suggestion
    LayerNorm(sz, λ=identity; affine=true, ϵ=1fe-5)
```